sympy/physics/control/lti.py

```diff
@@ -249,6 +261,108 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dcgain(self):
+        """
+        Computes the low-frequency gain of the frequency response in the
+        limit as frequency approaches to zero. For systems with pure
+        integrator terms, DC gain is infinite.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 3, s**2 - 9, s)
+        >>> tf1.dcgain()
+        -1/3
+        >>> tf2 = TransferFunction(p**2, p - 3 + p**3, p)
+        >>> tf2.dcgain()
+        0
+        >>> tf3 = TransferFunction(a*p**2 - b, s + b, s)
+        >>> tf3.dcgain()
+        (a*p**2 - b)/b
+
+        """
+        m = Mul(self.num, Pow(self.den, -1, evaluate=False), evaluate=False)
+        return limit(m, self.var, 0)
+
+    def pole(self):
+        """
+        Returns the poles of a transfer function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction((p + 3)*(p - 1), (p - 1)*(p + 5), p)
+        >>> tf1.pole()
+        [-5, 1]
+        >>> tf2 = TransferFunction((1 - s)**2, (s**2 + 1)**2, s)
+        >>> tf2.pole()
+        [I, I, -I, -I]
+        >>> tf3 = TransferFunction(s**2, a*s + p, s)
+        >>> tf3.pole()
+        [-p/a]
+
+        """
+        chareq = Poly(self.den, self.var)
+        try:
+            return _roots(chareq, self.var)
+        except NotImplementedError:
+            raise ValueError("Could not compute poles of this transfer function.")
```

Comment:
I'm not sure this is worth changing the message or type. Probably this would be enough
```suggestion
        return _roots(chareq, self.var)
```
or if you really things the message helps
```suggestion
        try:
            return _roots(chareq, self.var)
        except NotImplementedError as e:
            raise NotImplementedError("Could not compute poles of this transfer function.") from e
```